### ISSUE TITLE
Reflect new gophercloud structures

### ISSUE THREAD

**Issue by kayrus** [terraform-providers/terraform-provider-openstack#508]
Corresponding issue for the https://github.com/gophercloud/gophercloud/issues/1332

@jtopjian which tools/commands do you use to update the vendor directory?

**Comment by jtopjian** [comment]
Vendoring is currently done with [`govendor`](https://github.com/kardianos/govendor). You'll want to do something like:

```shell
$ govendor fetch github.com/gophercloud/gophercloud/...@<commit hash>
```

Where commit hash is the commit hash of the changes you want to bring in. 

Let me know if you run into any issues.

**Comment by ozerovandrei** [comment]
Implemented in #509, thank you @kayrus!
